packages/babel-core/src/util.js

```diff
@@ -80,8 +78,8 @@ export function regexify(val: any): RegExp {
 
 export function arrayify(val: any, mapFn?: Function): Array<any> {
   if (!val) return [];
-  if (isBoolean(val)) return arrayify([val], mapFn);
-  if (isString(val)) return arrayify(list(val), mapFn);
+  if (typeof val === "boolean") return arrayify([val], mapFn);
+  if (typeof val === "string") return arrayify(list(val), mapFn);
 
```

Comment:
I prefer using 
```js
isBoolean(x)
```

rather than 
```js
typeof x === "boolean"
```

We can remove lodash but I would suggest keeping helper functions for this.